x/pairing/keeper/scores/geo_req.go

```diff
@@ -83,6 +83,9 @@ func CalcGeoCost(reqGeo planstypes.Geolocation, providerGeos []planstypes.Geoloc
 }
 
 func calculateCostFromLatency(latency uint64) math.Uint {
```

Comment:
how can this be zero?
create a unitest that checks that there are no 0 in the table

Reply:
done. see https://github.com/lavanet/lava/commit/69fd139f8c489583fca76490a1f2b5aee4978a59